micro-ui/web/micro-ui-internals/packages/css/CHANGELOG.md

```
@@ -0,0 +1,141 @@
+
+## [1.8.2-beta.1]
+- Formatted changelog file.
+
+## [1.8.1-beta.11]
+- Updated styles for language selection dropdown options card.
+
+## [1.8.1-beta.10]
+- Fixed login header alignment.
+
+## [1.8.1-beta.8]
+- Used a new Primary constant color "#c84c0e".
```

Markdownlint

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)